lib/gems/pending/util/MiqSshUtil.rb

```diff
@@ -223,6 +288,18 @@ def temp_cmd_file(cmd)
     end
   end
 
+  # Shortcut method that creates and yields an MiqSshUtil object, with the +host+,
+  # +remote_user+ and +remote_password+ options passed in as the first three
+  # params to the constructor, while the +su_user+ and +su_password+ parameters are
+  # automatically set the :su_user and :su_password options. Remaining options are
```

Comment:
automatically se**n**t?  There's a typo or grammatical error here.

Reply:
I think the "are" is supposed to be removed. I reworded this sentence a couple times and it got a little goofed up.